### ISSUE TITLE
Privacy Policy and Terms of Service?

### ISSUE THREAD

**Issue by afeld** [prose/prose#961]
Hey folks-
@18F is interested in using http://prose.io, but unfortunately isn't able to do so without a Privacy Policy and a Terms of Service. Does that exist somewhere? If not, would it be something you'd be willing to add? Thanks!

**Comment by dereklieu** [comment]
Yeah definitely. We don't have a formal one but I can work on it.
